benchmarks/performancepoetry/src/main/java/com/squareup/benchmarks/performance/poetry/PerformancePoemWorkflow.kt

```diff
@@ -0,0 +1,244 @@
+import com.squareup.benchmarks.performance.poetry.views.BlankScreen
+import com.squareup.sample.container.overviewdetail.OverviewDetailContainer
+import com.squareup.sample.container.overviewdetail.OverviewDetailScreen
+import com.squareup.sample.poetry.PoemWorkflow
+import com.squareup.sample.poetry.PoemWorkflow.ClosePoem
+import com.squareup.sample.poetry.SimulatedPerfConfig
+import com.squareup.sample.poetry.StanzaListWorkflow
+import com.squareup.sample.poetry.StanzaListWorkflow.NO_SELECTED_STANZA
+import com.squareup.sample.poetry.StanzaRendering
+import com.squareup.sample.poetry.StanzaWorkflow
+import com.squareup.sample.poetry.StanzaWorkflow.Output.CloseStanzas
+import com.squareup.sample.poetry.StanzaWorkflow.Output.ShowNextStanza
+import com.squareup.sample.poetry.StanzaWorkflow.Output.ShowPreviousStanza
+import com.squareup.sample.poetry.StanzaWorkflow.Props
+import com.squareup.sample.poetry.model.Poem
+import com.squareup.workflow1.Snapshot
+import com.squareup.workflow1.StatefulWorkflow
+import com.squareup.workflow1.Worker
+import com.squareup.workflow1.WorkflowAction
+import com.squareup.workflow1.WorkflowAction.Companion.noAction
+import com.squareup.workflow1.action
+import com.squareup.workflow1.runningWorker
+import com.squareup.workflow1.ui.AndroidViewRendering
+import com.squareup.workflow1.ui.BuilderViewFactory
+import com.squareup.workflow1.ui.ViewFactory
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.backstack.BackStackScreen
+import com.squareup.workflow1.ui.backstack.toBackStackScreen
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+
+/**
+ * Renders a [Poem] as a [OverviewDetailScreen], whose overview is a [StanzaListRendering]
+ * for the poem, and whose detail traverses through [StanzaRendering]s.
+ */
+public class PerformancePoemWorkflow(
+  private val simulatedPerfConfig: SimulatedPerfConfig = SimulatedPerfConfig.NO_SIMULATED_PERF,
+  private val isLoading: MutableStateFlow<Boolean>
+) : PoemWorkflow, StatefulWorkflow<Poem, State, ClosePoem, OverviewDetailScreen>() {
+
+  public sealed class State {
+    // N.B. This state is a smell. We include it as possible in order to be able to config similar
+    // to real-world usage. It should be folded into [NoSelection] at the very least.
+    public object Initializing : State()
+    public data class ComplexCall(
+      val payload: Int
+    ) : State()
+
+    public data class Selected(val stanzaIndex: Int) : State()
+  }
+
+  @OptIn(WorkflowUiExperimentalApi::class)
+  public data class PoemRenderingWithLoading(
+    val screen: OverviewDetailScreen,
+    // This won't be used if we are reporting through Output.
+    val isLoading: Boolean = false
+  ) : AndroidViewRendering<PoemRenderingWithLoading> {
+    // Note that this ViewFactory does not handle the loading state at all. It just ignores it.
```

Comment:
Maybe give a hint about where it _is_ handled?

Reply:
This was dead code that I no longer used.